test/jdk/java/nio/channels/FileChannel/FileExtensionAndMap.java

```diff
@@ -117,6 +125,29 @@ public static void main(String args[]) throws Exception {
         }
     }
 
+    private static void checkRequiredDiskSpace(String destinationFolder) throws IOException {
+        Path destinationPath = Path.of(destinationFolder);
+        Path tmpDir = Path.of(TMPDIR);
+
+        long totalDiskSpaceNeeded = FILESIZE * PARALLELISM;
+
+        if (Files.getFileStore(tmpDir).name().equals(Files.getFileStore(destinationPath).name())) {
+            totalDiskSpaceNeeded *= 2; // writing and copying to same FS
+            long usableDiskSpace = Files.getFileStore(tmpDir).getUsableSpace();
+            if (usableDiskSpace < totalDiskSpaceNeeded) {
+                throw new SkippedException("Insufficient disk space on " + TMPDIR + ". Test requires: " + totalDiskSpaceNeeded + ". Available on disk: " + usableDiskSpace);
+            }
```

Comment:
Can you break. these overfly long lines so that future side-by-side diffs are possible?

Reply:
Sure, done.